code/ui/.storybook/main.ts

```diff
@@ -3,14 +3,17 @@ import type { StorybookConfig } from '../../frameworks/react-vite/dist';
 const config: StorybookConfig = {
   stories: [
     {
-      directory: '../manager/src/**/',
-      files: '*.stories.@(ts|tsx|js|jsx|mdx)',
-      titlePrefix: 'Manager',
+      directory: '../manager/src',
+      // these titles uses a special unicode character for the slash as a workaround for https://github.com/storybookjs/storybook/issues/9761
+      titlePrefix: '@storybook⁄ui',
```

Comment:
I couldn't even see that `/` in your screenshot, and to be honest I think it's fine without it